lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -283,7 +283,7 @@ def test_get_html_track(self):
             'id': self.item_descriptor.location.html_id(),
             'metadata': '',
             'track': None,
-            'transcript_download_format': 'srt',
+            'transcript_download_format': u'srt',
```

Comment:
These dictionaries are already an unholy mess of unicode and bytes.  I don't think changing one will cause an issue.  Note: these are being serialized, and the resulting strings compared, so it's literally the leading `u'` comparing unequal to the leading `'` on this field.